package.json

```diff
@@ -25,7 +25,7 @@
     "update": "ng update @nrwl/workspace",
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
-    "stamp": "nx run xlayers:stamp",
+    "stamp": "node scripts/stamp.js",
```

Comment:
```suggestion
    "stamp": "node tools/stamp.js",
```

Reply:
Could you change this to the suggested change?

Reply:
Oops sorry for the mistake, I shouldn't push code late on a sunday night! :) Fixed.